liberum_core/src/node/mod.rs

```diff
@@ -232,7 +233,22 @@ impl Node {
                 })
                 .await?;
 
-            return recv.await?.map_err(|e| e.into());
+            let DIAL_TIMEOUT = Duration::from_secs(3);
```

Comment:
Why use upper case, it is not constant.

Reply:
Actually, as you can see there is no `mut`, so it is a constant :nerd_face: 
https://github.com/Liberum-Neto/liberum-neto/pull/53/commits/aeb075baf393028a6a02a59d951f5a394f2c2598